naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/SangriaGraphQlSchemaBuilder.scala

```diff
@@ -66,7 +66,13 @@ class SangriaGraphQlSchemaBuilder(
       resourceObject <- (try {
         val resourceName = ResourceName(
           resource.name, resource.version.getOrElse(0L).toInt).identifier
-        generateLookupTypeForResource(resourceName)
+        if (!resource.handlers.exists(_.kind == HandlerKind.GET) ||
```

Comment:
i'd add a comment here explaining this logic (it took me a few times to understand what this was checking, and i knew what this logic was supposed to be 😄 )